lib/ansible/module_utils/common/collections.py

```diff
@@ -67,7 +67,8 @@ def difference(self, subtractive_iterable):
 
 def is_string(seq):
     """Identify whether the input has a string-like type (inclding bytes)."""
-    return isinstance(seq, (text_type, binary_type))
+    # AnsibleVaultEncryptedUnicode inherits from Sequence, but is expected to be a string like object
+    return isinstance(seq, (text_type, binary_type)) or getattr(seq, '__ENCRYPTED__', False)
```

Comment:
I was wondering what was the reason to check for the `__ENCRYPTED__` attribute instead of `isinstance(..., AnsibleVaultEncryptedUnicode)` until I realized this is in module_utils and we can't import things from outside of module_utils.